packages/code-du-travail-frontend/src/outils/common/Wizard.js

```diff
@@ -77,7 +77,7 @@ function Wizard({
                 )}
                 <StepItems activeIndex={stepIndex} items={stepItems} />
                 <StepWrapper narrow noPadding>
-                  <Step form={form} />
+                  <Step form={form} dispatch={dispatch} />
```

Comment:
this looks a very wrong idea :/ in terms of separation of concern

Reply:
Si tu as une meilleure idée je suis preneur mais là dans l'immédiat, j'ai une condition en fonction d'une CCN bien spécifique qui permet d'ajouter / enlever une step et je ne pense pas que ce soit une bonne idée que soit une bonne idée de mettre ça dans l'index du calculateur, alors que dans une step ça me paraît plus raisonnable. Mais bon je n'ai pas d'idée arrêtée là dessus ! N'hésite pas à proposer autre chose

Reply:
oui on pourrait adapter le fonctionnement de Condition pour filtrer les step au fur a mesure qu'on avance